packages/perseus/src/validation.types.ts

```diff
@@ -177,9 +177,14 @@ export type PerseusPlotterRubric = PerseusPlotterWidgetOptions;
 
 export type PerseusPlotterUserInput = ReadonlyArray<number>;
 
-export type PerseusRadioRubric = PerseusRadioWidgetOptions;
+export type PerseusRadioRubric = {
+    // The choices provided to the user.
+    choices: ReadonlyArray<PerseusRadioChoice>;
+};
 
 export type PerseusRadioUserInput = {
+    /*TODO(LEMS-2541): countChoices seems to be necessary for rendering the
+       question, not scoring. Maybe move to renderProps?*/
     countChoices?: boolean;
```

Comment:
So one way to look at the overall problem of RenderProps vs UserInput vs Rubric: is the thing in question used in the validator? Since `countChoices` isn't in the validator (or any of the functionality that the validator uses), it can't be in the Rubric (which is stuff we need from KA to validate the answer) or UserInput (which is stuff we need from the leaner to validate the answer). It's rendering logic, so RenderProps.

Then like `numCorrect` is a weird one:

- Is there anything the learner can do to affect `numCorrect`? No, so it's not UserInput.
- Is `numCorrect` something KA determines and is it used to score? Yes, so it's part of the Rubric.
- Is `numCorrect` something that determines how we render the component? Yes, so it needs to be part of RenderProps.

There's some overlap between all of this, but that's okay. What's important (for this leg of the work) is that we know exactly what shape the Rubric and UserInput is.

Reply:
I went through the code to see where removing `countChoices` from Radio's user input would require a refactor, but didn't see it. I asked Jeremy to follow-up on this today so you have coverage while I'm gone.

Reply:
I think I was getting some weird errors the first time, but I was able to follow through with removing `countChoices` from the `UserInput` type, so that's been added here.